content/architecture/access-control.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Access Control
+linktitle: Access Control
+aliases: [/rbac/]
+description: Managing Access Control
+date: 2017-02-01
+publishdate: 2017-02-01
+lastmod: 2017-02-01
+menu:
+  docs:
+    parent: "architecture"
+    weight: 101
+weight: 110
+sections_weight: 101
+draft: false
+toc: true
+---
+
+Jenkins X uses a Role Based Access Control (RBAC) policies to control access to its various resources.  The enforcement of the policies is provided by [Kubernetes' RBAC support](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
+
+As a given [Team](/about/features/#teams) can have a number of [Environments](/about/features/#environments) (e.g. Dev, Staging, Production) along with dynamic [Preview Environments](/developing/preview/) it can be a challenge keeping all the [`Role`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) and [`RoleBinding`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) resources from [Kubernetes RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) in sync with all the various namespaces and members of your team.
```

Comment:
```suggestion
As a given, [Teams](/about/features/#teams) can have a number of [Environments](/about/features/#environments) (e.g., Dev, Staging, Production) along with dynamic [Preview Environments](/developing/preview/). It can be a challenge keeping all the [`Role`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) and [`RoleBinding`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) resources from [Kubernetes RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) in sync with all the various namespaces and members of your team.
```

Reply:
I'm not convinced about how this reads, the two are linked  A causes B so should not be separate.